integration/mediation-tests/tests-patches/src/test/resources/artifacts/ESB/passthru/transport/httpaccesslogs/access-log.properties

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we update the year?

Reply:
Thanks. Updated

Reply:
Great but we are living in 2019 shall we up another year :)

Reply:
Oops will change that :D